realm/realm-library/src/androidTest/java/io/realm/BulkInsertTests.java

```diff
@@ -954,4 +954,29 @@ public void insertOrUpdate_collection_notInTransaction() {
     public void insertOrUpdate_object_notInTransaction() {
         realm.insert(new AllTypes());
     }
+
+
+    @Test
+    public void insertOrUpdate_fromOtherRealm() {
```

Comment:
@cmelchior this is a very odd remote tracking thing because this is for https://github.com/realm/realm-java/issues/3732 which was fixed in https://github.com/realm/realm-java/pull/3755